Content/Maatregelen/ProjectOrganisatie/M13 - Dekking ISO-25010/Maatregel.md

```diff
@@ -1,4 +1,6 @@
-De door de projectenorganisatie vastgestelde kwaliteitsnormen dekken een significant deel van de ISO-25010-kwaliteitskenmerken af.
+De standaard ISO/IEC 25010:2011, kortweg "ISO-25010", biedt een model voor het beschrijven van productkwaliteit. Kwaliteitseigenschappen zijn voorzien van een naam, definitie en classificatie. ISO-25010 dekt een breed spectrum van kwaliteitseigenschappen af.
+
+Voor specificatie en documentatie van vereiste en gewenste kwaliteitseigenschappen, de niet-functionele eisen, maken projecten gebruik van de terminologie uit ISO-25010. Projecten gebruiken ISO-25010 als controlemiddel voor de mate waarin relevante kwaliteitseigenschappen van het op te leveren eindproduct zijn afgedekt en binnen scope geplaatst.
```

Comment:
Suggestie: "Projecten gebruiken ISO-25010 om te controleren of alle relevante kwaliteitseigenschappen van het op te leveren eindproduct worden meegenomen in de ontwikkeling en/of onderhoud van het product."